akka-epl/src/main/scala/com/epl/akka_v1/HttpClient.scala

```diff
@@ -0,0 +1,85 @@
+package com.epl.akka_v1
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.marshalling.Marshal
+import akka.http.scaladsl.model.headers.BasicHttpCredentials
+import akka.http.scaladsl.model.{HttpMethods, HttpRequest, RequestEntity, headers}
+import akka.http.scaladsl.unmarshalling.Unmarshal
+import akka.stream.ActorMaterializer
+import akka.util.ByteString
+import com.epl.akka.DocumentType
+
+import scala.concurrent.Future
+
+object HttpClient {
+
+  implicit val system = ActorSystem()
+  implicit val materializer = ActorMaterializer()
+  implicit val ec = system.dispatcher
```

Comment:
Don't create a separate actor system for this, instead, make this a class that takes the system and materializer as constructor parameters, and pass in the system of the actor where you use it.